src/app/functions/server/getRelevantGuidedSteps.ts

```diff
@@ -0,0 +1,241 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Session } from "next-auth";
+import { LatestOnerepScanData } from "../../../db/tables/onerep_scans";
+import { SubscriberBreach } from "../../../utils/subscriberBreaches";
+import { BreachDataTypes, HighRiskDataTypes } from "../universal/breach";
+
+export type InputData = {
+  user: Session["user"];
+  countryCode: string;
+  latestScanData: LatestOnerepScanData | null;
+  subscriberBreaches: SubscriberBreach[];
+};
+
+// Note: the order is important; it determines in which order the user will be
+//       guided through the pages.
+export const stepLinks = [
```

Comment:
Question: Are there use cases where it would be easier to make this a two-dimensional array and group them per high-level step? Currently, I can only think of marking previous steps as ”done” being easier.
<img width="525" alt="image" src="https://github.com/mozilla/blurts-server/assets/13835474/c741a5c3-dcdb-42b5-bff4-e7e366e043be">

Reply:
Hmm, not sure. I think I'd rather keep the data structure simple, and potentially add a helper function to operate on the parts that apply to specific steps later, if we do encounter that use case often.